src/main/java/haveric/recipeManager/tools/Version.java

```diff
@@ -52,6 +55,19 @@ private static boolean supports1_11() {
 
         return supports;
     }
+    private static boolean supports1_10() {

```

Comment:
Probably isn't. I had some thoughts earlier on to go back and look at what was changed in 1.10 vs. 1.11 and split those out. Probably won't. I can remove it.